src/_locales/pl-PL.json

```diff
@@ -42,9 +42,9 @@
 
   "modal-confirmTray-title": "Chcieliśmy tylko uprzedzić",
   "modal-confirmTray-content": "Zamknięcie głównego okna aplikacji (czyli to, co właśnie próbowałeś zrobić) w rzeczywistości zminimalizuje odtwarzacz do systemowego paska narzędzi. Możesz zmienić to ustawienie w oknie Ustawień Aplikacji.",
-  "modal-confirmOpenPort-content": "Playback API nie mogło zostać uruchomione, ponieważ jest ono blokowane przez Windows Firewall. Możemy to dla Ciebie odblokować, ale ostrzegamy, że może to wymagać uprawnień administratora",
+  "modal-confirmOpenPort-content": "Nie można udzielić dostępu do API, ponieważ operacja została zablokowana przez Windows Firewall. Możemy go dla Ciebie odblokować, ale ostrzegamy, że może to wymagać uprawnień administratora",
   "modal-confirmUpdate-title": "Dostępna aktualizacja",
-  "modal-confirmUpdate-content": "Hurra, wygląda na to, że dostępna jest nowa wersja Google Play Music Desktop Player. Ta aktualizacja powinna przynieść całe mnóstwo nowych funkcji i poprawek. Rekomendujemy zaktualizowanie aplikacji najszybciej jak to możliwe, żeby zapewnić najlepsze wrażenia z korzystania z niej.<br />Mamy nadzieję, że polubisz nową wersję!",

```

Comment:
Woah

Almost let this one slip through.  The `<br />` is required and must remain in this string